### DISCUSSION TITLE
What's the default colors configurations of `--color=always`?

### DISCUSSION THREAD

**Discussion by linrongbin16** [BurntSushi/ripgrep#2605] [Q&A] [answered]
Hi, what's the default colors configurations of `--color=always`?

**Comment by BurntSushi** [comment]
The default is `--color=auto`.

**Reply by linrongbin16** [reply]
Thanks for reply, I mean, the `purple` for file path, `green`  for line number, etc things.

For example, I'm running command in my working directory:

```bash
rg --column -n --no-heading --color=always -S -- 'fzfx'
```

I didn't configure anything for `rg` (no configuration file, no `.ignore` files)

It gives me:

<img width="1910" alt="image" src="https://github.com/BurntSushi/ripgrep/assets/6496887/df26e410-a9ca-4c7d-8abb-03c68745ff0d">

It looks like the **default** colors configuration of rg is:

`purple` for file path (e.g. `autoload/fzfx/nvim.vim`), `green` for line number, `red` for matching text.

So is there some document about this thing?

**Reply by BurntSushi** [reply, marked as answer]
Defaults are defined here: https://github.com/BurntSushi/ripgrep/blob/d596f6ebd035560ee5706f7c0299c4692f112e54/crates/printer/src/color.rs#L14

There is no documentation describing the default because it's not clear why it should be documented. Why are you asking?

The docs for the `--colors` flag explain how to change the colors.

**Reply by linrongbin16** [reply]
Because I'm developing a Neovim plugin, and I want to render some lines exactly like rg's style in it.

Thanks for the code link!
